src/AutoLaunchLinux.coffee

```diff
@@ -25,18 +25,18 @@ module.exports =
         return fileBasedUtilities.createFile {
             data
             directory: @getDirectory()
-            filePath: @getFilePath appName
+            filePath: @getFilePath appName.toLowerCase()
         }
 
 
     # appName - {String}
     # Returns a Promise
-    disable: (appName) -> fileBasedUtilities.removeFile @getFilePath appName
+    disable: (appName) -> fileBasedUtilities.removeFile @getFilePath appName.toLowerCase()
 
 
     # appName - {String}
     # Returns a Promise which resolves to a {Boolean}
-    isEnabled: (appName) -> fileBasedUtilities.isEnabled @getFilePath appName
+    isEnabled: (appName) -> fileBasedUtilities.isEnabled @getFilePath appName.toLowerCase()
```

Comment:
Why are you lowercasing the appName?

Would be good to make a test with different app name cases to test that this still works

Reply:
Could you pull from master so tests will run for your commits?

Reply:
> Why are you lowercasing the appName?
> 
> Would be good to make a test with different app name cases to test that this still works

Good question. I've been going on and off about it. There is nothing requiering the filename to be in lowercase. And we can also go in favor with the fact that we should honor how the appName sent to us. On the other hand, most apps go with lowercase filename for Desktop file. This wouldn't be a problem if we were to support a parameter for the appName and one for the Desktop file's name, but we are not offering this option at this point.

For testcases, I think we should at least add a test where the appName could have a space in it, which should be a supported scenario.

Reply:
> Could you pull from master so tests will run for your commits?

You want me to pull in my PR or you want me to pull it out? Testcases were succeeding in my fork.

Reply:
I chose to respect case by: https://github.com/Teamwork/node-auto-launch/pull/120/commits/2483d2b53388d10e98d4633daee09e4282755761

Reply:
@4ver I'll need your review to be able to pull in the series. Your comments are welcomed.

Reply:
> I've been going on and off about it. There is nothing requiering the filename to be in lowercase. And we can also go in favor with the fact that we should honor the appName sent to us. On the other hand, most apps go with lowercase filename for Desktop file. 

Not sure i get you. On linux, the filesystem is case-sensitive so there could technically be two files with the same name but with different cases. Probably best to go with what's been passed as appName?

> You want me to pull in my PR or you want me to pull it out? Testcases were succeeding in my fork.

Ah sorry! I haven't set it up to run tests on pull requests. Just commits. I checked your fork and they are indeed passing.

Could you add some tests to cover the case with a space and with an app name that's not all lowercase please.

Reply:
@4ver Latest commit should take care of the requested testcases. And we make sure to remove any files we may have created for the tests' purpose.